3_extracting_features/README.md

```diff
@@ -0,0 +1,104 @@
+
+**`NF1_cyto_config.json`:**
+- `dataset: locations: box_size: 96 -> dataset: locations: box_size: 256` This change expands the size of the box around each cell that DeepProfiler interprets. 
+This expansion attempts to capture as much of the cytoplasm as possible (this will be benchmarked in the future to assess the best box size).
+
+## Step 1: Setup Feature Extraction Environment
+
+### Step 1a: Create Feature Extraction Environment
+
+```sh
+# Run this command to create the conda environment for feature extraction
+conda env create -f 3.NF1_feature_extraction_env.yml
+```
+
+### Step 1b: Activate Feature Extraction Environment
+
+```sh
+# Run this command to activate the conda environment for Deep Profiler feature extraction
+
+conda activate 3.feature-extraction-NF1
+```
+
+## Step 2: Install DeepProfiler
+
+### Step 2a: Clone Repository
+
+Clone the DeepProfiler repository into 3_extracting_features/ with 
+
+```console
+# Make sure you are located in 3_extracting_features/
+cd 3_extracting_features/
+git clone https://github.com/cytomining/DeepProfiler.git
+```
+
+### Step 2b: Install Repository
+
+Install the DeepProfiler repository with
+
+```console
+# Make sure you are located in DeepProfiler/ to install
+cd DeepProfiler/
+pip install -e .
+```
+
+### Step 2c: Complete Tensorflow GPU Setup
+
+Based on previous projects within the lab, we found using Tensorflow GPU when using DeepProfiler improves performance. To setup, follow [these instructions](https://www.tensorflow.org/install/pip#3_gpu_setup).
+I use Tensorflow GPU while processing NF1 data.
+
+## Step 3: Define Project Paths
+
+Inside the notebook [compile_DP_projects.ipynb](compile_DP_projects.ipynb), the variables `nuc_project_path` and `cyto_project_path` need to be changed to reflect the desired object DeepProfiler project locations.
+
+## Step 4: Compile DeepProfiler Project
+
+In order to profile features with DeepProfiler, a project needs to be set up with a [certain file structure and files](https://github.com/cytomining/DeepProfiler/wiki/2.-Project-structure).
+
+In [compile_DP_projects.ipynb](compile_DP_projects.ipynb), the necessary project structure is created using the functions from [DPutils.py](DPutils.py).
+
+The config files ([NF1_nuc_config.json](DP_files/NF1_nuc_config.json)/[NF1_cyto_config.json](DP_files/NF1_cyto_config.json)) are copied to their corresponding projects and the pretrained model ([efficientnet-b0_weights_tf_dim_ordering_tf_kernels_autoaugment.h5](DP_files/efficientnet-b0_weights_tf_dim_ordering_tf_kernels_autoaugment.h5)) to both projects.
+Both of these files are located within the [DP_files folder](DP_files/) for reference.
+
+An `index.csv` file needs to be compiled as it necessary for DeepProfiler to load each image, which is created using the DP_files. Using the index.csv, the locations are compiled (in project/input/locations) and these csv files are necessary for DeepProfiler to find the single cells in each image.
```

Comment:
```suggestion
We need to compile an `index.csv` file as it necessary for DeepProfiler to load each image.
We create this using the DP_files.
Using the index.csv, we compile the locations (in project/input/locations), which are necessary csv files for DeepProfiler to find the single cells in each image.
```

We should try our best to use active voice instead of passive voice.

I didn't highlight all instances of passive voice in the PR.

Reply:
also, I don't know what "which is created using the DP_files" means so I don't know if my translation works!